lib/segment/src/index/struct_payload_index.rs

```diff
@@ -156,11 +160,30 @@ impl StructPayloadIndex {
         let config = if config_path.exists() {
             PayloadConfig::load(&config_path)?
         } else {
+            // ToDo(mmap-paylaod-index): uncomment before minor release
```

Comment:
Did you manually test the `skip_rocksdb` part?

Reply:
there is one green CI test pass with this option enabled and one (latest) where this option is disabled